### PR TITLE
Style the scrollbar in the side menu

### PR DESCRIPTION
**Pull request by dmandrade**
A simple style to improve the look of the side menu with a scrollbar.

NOTE: the scrollbar appears only on the sidebar hover

image
image

### CODE REVIEW

**Comment by awcodes** [comment]
Code looks good to me. 

I think it could be a problem on computers that have scrollbars always set to show (a11y issue). They may not support the styling then the scrollbar will probably creep into the icons.

Also an 8px wide bar is going to be hard to grab to use without a scroll wheel. But it is about the size of the default scrollbar on Mac Chrome. So probably not a big deal.

**Comment by zepfietje** [comment]
I think the dark mode color needs tweaking.

**Comment by danharrin** [comment]
Make changes to the PR if you want

File: packages/admin/resources/css/app.css [hunk 1]
```diff
@@ -49,3 +49,23 @@
         top: -1px;
     }
 }
+
+@layer utilities {
```
Comment (zepfietje) [inline packages/admin/resources/css/app.css:53]: I think the `utilities` layer is wrong for this goal.

**Review by zepfietje — CHANGES_REQUESTED** [review]

File: packages/admin/resources/css/app.css (hunk 1 above)
Comment (awcodes) [reply packages/admin/resources/css/app.css:53]: I agree. Should either be in base or components.

**Comment by zepfietje** [comment]
I think we should hold off for now and not customize the scrollbar by default in Filament admin. Browser support hasn't reached the point where we can achieve consistent styling between browsers.

We might revisit this in the future, but I'd say no for now.
@awcodes, @pxlrbt, @danharrin agree?

**Comment by danharrin** [comment]
Agreed

**Pull request closed without merging**